src/ui/public/domLocation.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function DomLocationProvider($window) {
```

Comment:
Just curious, why do you create this module instead of using something like `$location`?